### PR TITLE
Add Connector page, update readme, update SLA?

### PR DESCRIPTION
**Pull request by tayloramurphy**

### CODE REVIEW

File: docs/connectors.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+## Connector Maintainers
+
+* **Official** -  Official connectors and tools are built, maintained, supported, and tested by Meltano.
+* **Partner** - Partner connectors and tools are built, maintained, supported, and tested by a partner organization of Meltano, including consultancies and other vendors.
+* **Community** - Community connectors and tools are built and maintained by the wider open source community.
+
+## Connector Quality
+
+* **Platinum** - Platinum connectors are built on the Meltano SDK and are maintained, supported, and tested by Meltano or our Partners. Additional SLAs are available for these connectors in Meltano Cloud. Contact us to learn more.
```
Comment (tayloramurphy) [inline docs/connectors.md:9]:
```suggestion
```

File: docs/connectors.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+## Connector Maintainers
+
+* **Official** -  Official connectors and tools are built, maintained, supported, and tested by Meltano.
+* **Partner** - Partner connectors and tools are built, maintained, supported, and tested by a partner organization of Meltano, including consultancies and other vendors.
+* **Community** - Community connectors and tools are built and maintained by the wider open source community.
+
+## Connector Quality
+
+* **Gold** - Gold connectors are built on the Meltano SDK, are actively used by many Meltano Cloud and open source users, and have broad coverage of data and features.
```
Comment (DouweM) [inline docs/connectors.md:9]: Self-managed, not open source, as Cloud usage is also largely open source:

```suggestion
* **Gold** - Connectors built on the Meltano SDK with high Meltano Cloud and self-managed usage, and broad coverage of data and features.
```

File: docs/connectors.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+* **Silver** - Silver connectors may be built on the Meltano SDK, but their open source and Meltano Cloud usage may be lower compared to a Gold connector.
```
Comment (DouweM) [inline docs/connectors.md:10]: @tayloramurphy So these are 1) SDK-based connectors with low to medium usage data (as SDK+high = Gold) and 2) non-SDK connectors with medium to high usage data (as non-SDK+low = Bronze)? I think that makes sense -- even with low usage data, we generally trust a SDK-based tap more than a non-SDK one. I think that can be more clear though:

```suggestion
* **Silver** - Connectors built on the Meltano SDK with low to medium Meltano Cloud and self-managed usage, and connectors not built on the Meltano SDK with high usage.
```

File: docs/connectors.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+* **Bronze** - Bronze connectors are not based on the Meltano SDK. We have some usage data for this connector in Meltano Core or Cloud.
```
Comment (DouweM) [inline docs/connectors.md:11]:
```suggestion
* **Bronze** - Connectors not built on the Meltano SDK with low to medium Meltano Cloud and self-managed usage.
```

File: docs/connectors.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+* **No Data** - Connectors for which we have no usage data. They likely have not been tested and the repository maintainers may be unresponsive.
```
Comment (DouweM) [inline docs/connectors.md:12]:
```suggestion
* **No Data** - Connectors for which we have no Meltano Cloud or self-managed usage data. These may be poorly tested or out of date, and repository maintainers may be unresponsive.
```

File: docs/connectors.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+
+
+### Connector Quality Matrix
+
+To give users a better understanding of the overall quality of a connector, we have the following matrix and guidance on attributes that affect the quality rating.
+
+|         | SDK-Based | Usage Data    | Maintainer         | Repo Activity  |
+|---------|-----------|---------------|--------------------|----------------|
+| Gold    | Yes       | > 5 Projects  | Meltano or Partner | High           |
```
Comment (DouweM) [inline docs/connectors.md:21]: @tayloramurphy Wouldn't a _community_-maintained connector "built on the Meltano SDK, are actively used by many Meltano Cloud and open source users, and have broad coverage of data and features" also be Gold? 

```suggestion
| Gold    | Yes       | > 5 Projects  | Official, Partner, or Community | High           |
```

File: docs/connectors.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+| Silver  | Possibly  | >= 1 Projects | Community          | Medium on Repo |
+| Bronze  | No        | >= 1 Projects | Community          | Low            |
+| No Data | No        | Unkonwn       | Community          | Unknown        |
+
+<!-- Generated from https://www.tablesgenerator.com/markdown_tables -->
```
Comment (DouweM) [inline docs/connectors.md:26]:
```suggestion
```

File: docs/connectors.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+
+For all connectors we may apply feedback we get from the community or our judgement to adjust the quality indicator of a connector.
+
+Please reach out to us to learn more about Platinum-level support for Gold connectors.
```
Comment (DouweM) [inline docs/connectors.md:30]:
```suggestion
Please reach out to us to learn more about Platinum-level support for Gold connectors on Meltano Cloud.
```

File: docs/connectors.md [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+
+**Are these connectors production ready?**
+
+Gold connectors are production ready. Silver connectors are very likely ready for production. Bronze connectors could be ready for production, but assessing them for your use case is recommended.
```
Comment (DouweM) [inline docs/connectors.md:34]:
```suggestion
Gold connectors are production ready. Silver connectors are very likely production ready. Bronze connectors and connectors on which we have No Data could be production ready, but you'll want to assess this yourself, and modifications may be necessary.
```

File: docs/connectors.md [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+
+One additional factor is what stage of development the connector is in. We recommend [semantic versioning](https://semver.org/) for connector development. If a connector is at v1.0.0 or greater, then it is definitely production-grade. For connectors between versions 0.1.0 and 1.0.0, they could be production ready and a number of users could be using them actively. But it is worth considering your own use case and whether it has the streams and features you need. If the connector is <0.1.0 then it still in development and is likely not ready for production.
```
Comment (DouweM) [inline docs/connectors.md:36]:
```suggestion
One additional factor is what stage of development the connector is in. We recommend [semantic versioning](https://semver.org/) for connector development. If a connector is at v1.0.0 or greater, then it is considered production ready by its creator. For connectors between versions 0.1.0 and 1.0.0, they could be production ready and a number of users could be using them actively, but it is worth considering your own use case and whether it has the streams and features you need. If the connector is <0.1.0 then it is still in development and likely not ready for production, but feedback and contributions are welcomed to make it so.
```

File: docs/connectors.md [hunk 11]
```diff
@@ -0,0 +1,40 @@
...
+
+**Why is Silver the maximum rating for connectors not based on the Meltano SDK?**
+
+We find that many connectors not based on the SDK may not have the full feature set of SDK-based connectors. Even for connectors that have high usage, the fact that they are not based on the SDK means that it's much harder to validate that they conform to the full Singer specification. It also means they are harder to maintain and improve. They also miss out on improvements that are added regularly to the Meltano SDK.
```
Comment (DouweM) [inline docs/connectors.md:40]:
```suggestion
Connectors that are not based on the Meltano SDK typically do not use all the features and performance optimizations available in the Singer specification and the SDK implementation thereof. Even if they have high usage, their quality and feature and data coverage are harder to asses, they will not benefit from future improvements to the Singer spec and SDK, and that they are significantly harder to maintain and contribute to.
```

File: docs/sla.md [hunk 12]
```diff
@@ -1,3 +1,3 @@
-# Meltano Cloud Service Level Agreement (SLA)
+# Meltano Cloud Platform Service Level Agreement (SLA)
 
-Todo
+# Meltano Cloud Platinum Connector Service Level Agreement (SLA)
```
Comment (DouweM) [inline docs/sla.md:3]: This is the main SLA page we'll link from the contract, _not_ specific to Platinum Connector Support, so please roll back this change!

**Review by DouweM — CHANGES_REQUESTED** [review]

File: docs/connectors.md (hunk 3 above)
Comment (tayloramurphy) [reply docs/connectors.md:10]: @DouweM Yes - you've got the summary mostly right. Many of the transferwise connectors should be Silver - non-SDK based but high usage. An SDK-based connector _could_ be Silver depending on usage and coverage of streams. But that can be adjusted based on our judgement.

File: docs/connectors.md (hunk 6 above)
Comment (tayloramurphy) [reply docs/connectors.md:21]: @DouweM I think that's fair. When I set this up originally I was thinking Gold would be a bonus for Partner-maintained connectors, but it's not necessary at this stage.

**Comment by tayloramurphy** [comment]
Closing this as i have a more complete PR in https://github.com/meltano/meltano/pull/7428 I'm pretty happy with it as a first iteration.

**Pull request closed without merging**